src/React/Resources/assets/src/render_controller.ts

```diff
@@ -27,6 +27,10 @@ export default class extends Controller {
 
         this._dispatchEvent('react:connect', { component: this.componentValue, props: props });
 
+        if (!this.componentValue) {
+            throw new Error('No component specified for react controller');
```

Comment:
```suggestion
            throw new Error('No component specified.');
```

Shortening, just because this isn't an error that should ever happen, but this full text will ship to production.